app/javascript/lib/validation/io.js

```diff
@@ -9,7 +9,10 @@
  * [1] https://stackoverflow.com/a/47976589
 */
 
-export const DEFAULT_CHUNK_SIZE = 1024 * 1024 // 1 MiB
+export const MiB = 1024 * 1024 // 1 MiB
+
+export const GiB = MiB * 1000 // 1 GiB
```

Comment:
I found these export labels confusing (but that might just be that I've never actually used Mebibytes before, instead of MB).  I might have named these ONE_MiB or 1_MiB.  Is there a reason we are using the much more obscure Mebibyte than a MB?
Also, the things I'm looking at online say that a GiB is 1024^3 instead of MiB * 1000.

Reply:
Great spot with GiB.  Fixed.

It might be worth reevaluating, but I recall we began using binary prefixes (e.g. MiB) instead of decimal prefixes (e.g. MB) because that's what's most often used in network transfers, file storage, and/or quotas.  

However, those binary prefixes are actually _displayed_ in UIs as decimal prefixes, which can be very confusing.  So we opted to use the correct (and admittedly rather technical) binary prefix in our code.